src/app/(dashboard)/repositories/new/page.tsx

```diff
@@ -62,7 +62,10 @@ export default function CreateTeamPage() {
     }
   };
 
-  const handleOpenRepoDialog = async (repo: GithubRepoDto) => {
+  const handleOpenRepoDialog = async (
+    repo: GithubRepoDto,
+    selected?: boolean,
```

Comment:
You sure this was intended? Why you changing it here on this page of the old structure?

Reply:
Oh I see, it's because of the card props yeah?

Have you tested if it's going to change the appearance of the old repository selection flow? Would be weird if when the user selects a repo, then the modal for confirmation opens and he cancels, if the repo continues selected there. 

If that happens might be better to duplicate the file for now as we are going to delete it anyway after we move to projects, or to add another flag for behaviour

Reply:
Tested, all good works like a glove